.github/workflows/terraform-cognito-test.yaml

```diff
@@ -48,6 +48,7 @@ jobs:
       uses: hashicorp/setup-terraform@v2
       with:
         terraform_version: 1.2.7
+        terraform_wrapper: false
```

Comment:
another issue is in cleanup step

`cd deployments/cognito/terraform` instead of `cd ../../deployments/cognito/terraform`

Reply:
thanks, added